test/e2e/tests/send-eth.spec.js

```diff
@@ -122,9 +122,6 @@ describe('Send ETH from inside MetaMask using fast gas option', function () {
         const inputValue = await inputAmount.getAttribute('value');
         assert.equal(inputValue, '1');
 
-        // Set the gas price
```

Comment:
Why is this e2e test change needed?

Reply:
Because we don't have the Slow/Medium/Fast inline buttons after this PR.

Reply:
nevermind, this was probably a holdover from when I was juggling branches.